### PR TITLE
Moved eslint to "scripts"

### PR DESCRIPTION
**Pull request by travesty8**
Also Using pretty-quick in place of lint-staged

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -17,16 +17,24 @@ app.use(bodyParser.json());
...
+  );
   io.emit(EVENTS.USER_CONNECTED, { dateConnected });
 
-  socket.on("disconnect", function() {
+  socket.on('disconnect', function() {
     const dateDisconnected = new Date();
 
-    console.log(`${chalk.yellow("Event:")} ${chalk.red(EVENTS.USER_DISCONNECTED)} ${socket.id} ${dateConnected}`);
+    console.log(
+      `${chalk.yellow('Event:')} ${chalk.red(EVENTS.USER_DISCONNECTED)} ${
```
Comment (ewendlick) [inline server.js]: I think the linter is behaving a little strangely here

File: server.js [hunk 2]
```diff
@@ -17,16 +17,24 @@ app.use(bodyParser.json());
  * 1. Register routes
  * 2. Start server
  */
-io.on("connection", (socket) => {
+io.on('connection', (socket) => {
   const dateConnected = new Date();
 
-  console.log(`${chalk.yellow("Event:")} ${chalk.blue(EVENTS.USER_CONNECTED)} ${socket.id} ${dateConnected}`);
+  console.log(
+    `${chalk.yellow('Event:')} ${chalk.blue(EVENTS.USER_CONNECTED)} ${
```
Comment (ewendlick) [inline server.js]: The linter is behaving a little strangely here

File: server.js (hunk 1 above)
Comment (travesty8) [reply server.js]: Yeah, it is a little weird. Lets increase the line width to 120 (from 80)

**Comment by travesty8** [comment]
Okay, increased to 120

**Pull request merged**